sdk/scheduler/src/main/java/com/mesosphere/sdk/offer/evaluate/placement/PlacementUtils.java

```diff
@@ -121,12 +131,38 @@ public static PlacementField getField(String fieldName) {
         }
     }
 
+    public static boolean hasZone(Protos.Offer offer) {
+        if (!offer.hasDomain()) {
+            return false;
+        }
+
+        Protos.DomainInfo domainInfo = offer.getDomain();
+        if (!domainInfo.hasFaultDomain()) {
+            return false;
+        }
+
+        Protos.DomainInfo.FaultDomain faultDomain = domainInfo.getFaultDomain();
+        if (faultDomain.hasZone()) {
+            return true;
+        }
+
+        return false;
+    }
+
     public static boolean placementRuleReferencesRegion(PodSpec podSpec) {
+        return placementRuleReferencesField(PlacementField.REGION, podSpec);
+    }
+
+    public static boolean placementRuleReferencesZone(PodSpec podSpec) {
```

Comment:
If it's expected that this will only be called against the parent ServiceSpec like it is in cassandra, this could be updated to accept the ServiceSpec and loop over the PodSpecs internally.